### PR TITLE
Begone DEFLECTION.

### PR DESCRIPTION
**Pull request by mishulinya**
## About The Pull Request

 Removes deflection temporarily as frok wanted it, lowers some block percentage for higher tier PA since it's so fucking high. Third, let me get rid of the other files that aren't pa related because github is being stupid. 

## Why It's Good For The Game

- Deleting deflection would be a nerf to PA and would stop them from being literally invincible.

## Pre-Merge Checklist
- [X] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [X] There are no new Runtimes that appear.
- [X] You documented all of your changes.

## Changelog
:cl:
del: Deflection.
/:cl:

### CODE REVIEW

**Comment by mishulinya** [comment]
Replaced the files, it's all good to go.

**Comment by mishulinya** [comment]
The Run Linters error is just missing output text file. I don't know what's up with that.

**Comment by hecksadecimal** [comment]
> 
> 
> The Run Linters error is just missing output text file. I don't know what's up with that.

image

**Comment by mishulinya** [comment]
> image

Lemme see which line is missing it. Gimme a minute.

**Comment by mishulinya** [comment]
> 
> 
> > The Run Linters error is just missing output text file. I don't know what's up with that.
> 
> image

It's fixed, i just forgot a empty line.

**Comment by hecksadecimal** [comment]
Yeah linter looks happy now.

File: code/modules/clothing/suits/power_armor.dm [hunk 1]
```diff
@@ -167,8 +167,8 @@
 	requires_training = TRUE
 	powered = TRUE
 	slowdown = 0.24
-	armor_block_chance = 60
-	deflection_chance = 10 //20% chance to block damage from blockable bullets and redirect the bullet at a random angle
+	armor_block_chance = 30
+	deflection_chance = 0 //20% chance to block damage from blockable bullets and redirect the bullet at a random angle
```
Comment (printer16) [inline code/modules/clothing/suits/power_armor.dm:171]: You can just remove all of the liens you set to 0 instead of leaving them in.

Comment (mishulinya) [reply code/modules/clothing/suits/power_armor.dm:171]: Frok wanted me to set the values to 0. That’s why i did that.

**Comment by printer16** [comment]
As I said in discord, the default value is 0 so it's best to just remove the lines all together.

**Comment by AdamElTablawy** [comment]
The developer of this PR left.

**Pull request closed without merging**